pkg/materialize/source_mysql.go

```diff
@@ -111,7 +117,9 @@ func (b *SourceMySQLBuilder) Create() error {
 		}
 		q.WriteString(`)`)
 	} else {
-		q.WriteString(` FOR ALL TABLES`)
+		if b.allTables {
+			q.WriteString(` FOR ALL TABLES`)
+		}
```

Comment:
Does this change the default behavior?

If someone didn't define all_tables and did not have a list of tables then upgraded to this version I think their source might be recreated with no tables?

Reply:
I should have probably started with that PR: https://github.com/MaterializeInc/terraform-provider-materialize/pull/775

This should be covered in the migration guide, tldr is: use a `ignore_changes` lifecycle block to ignore changes while also defining the tables using the new source table resources.